charts/incubator/euterpe/values.yaml

```
@@ -1,25 +1,57 @@
-env: {}
 image:
   pullPolicy: IfNotPresent
   repository: tccr.io/truecharts/euterpe
-  tag: latest@sha256:5be892d3559100899f7b36a9bc3894448414b53dd6af885f3606fa0316b6b44a
-persistence:
-  storagelibrary:
-    enabled: true
-    mountPath: /root/Music
+  tag: 1.5.2@sha256:5be892d3559100899f7b36a9bc3894448414b53dd6af885f3606fa0316b6b44a
+
 podSecurityContext:
   runAsGroup: 0
   runAsUser: 0
+
 securityContext:
   readOnlyRootFilesystem: false
   runAsNonRoot: false
+
+euterpe:
+  basic_authenticate: false
+  authentication:
+    user: ""
+    password: ""
+  libraries: ["/root/Music"]
```

```suggestion
  libraries: []
```
This will get ovewriten from questions